Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.py

```diff
@@ -4331,264 +4321,306 @@ def set_snapshot_labels(args):
     )
 
 
+def add_project_info_metadata(metadata):
+    """
+    Add or update project wide metadata.
+    :param metadata:  Each metadata entry is a key/value pair separated by ';' like so: key=abc,value=123;key=abc,value=123
+    """
+    project = SERVICE_ACT_PROJECT_ID
+    project_instance = get_compute().projects().get(project=project).execute()
+    fingerprint = project_instance.get('tags', {}).get('fingerprint')
+    items = parse_labels(metadata)
```

Comment:
how dose the error look to the user if the metadata was in a bad format?
I dont see a special handling. I think the user should get an error that shows the actual content and suggest the correct format.

Reply:
@yuvalbenshalom 
![image](https://user-images.githubusercontent.com/20818773/81496319-77878380-92bf-11ea-899b-822b8b48a097.png)